### PR TITLE
SEO

### PR DESCRIPTION
**Pull request by m-ata**

### CODE REVIEW

File: next.config.js [hunk 1]
```diff
@@ -16,6 +16,51 @@ const nextConfig = {
...
+    {
+      source: '/team/Head-of-Developer-Relations/1e365b6a-9df0-5eec-9762-a4b25f913d23',
+      destination: '/410?isJobExpired=true',
+      permanent: true,
+    },
+    {
+      source: '/team/Head-of-Engineering/6976dbe1-4155-5560-8c1b-16c894141956',
+      destination: '/410?isJobExpired=true',
+      permanent: true,
+    },
```
Comment (miedziak) [inline next.config.js:53]: todo: If you will get more and more 410 redirect, this flags like `isJobExpired` will start to be quite messy. I would suggest to create separate `410` files, one for docs redirects and second for 'team' redirects. So there will be: 
```jsx
    {
      source: '/posts/:slug*',
      destination: '/docs-410',
      permanent: true,
    },
...
    {
      source: '/team/Senior-Support-Engineer/914dd8e6-e94a-4a23-90db-d92d271d41e2',
      destination: '/jobs-410',
      permanent: true,
    },

```

Comment (miedziak) [inline next.config.js:53]: suggestion: I think you can group this job redirects to separate file and just extract it here, will be cleaner when reading this file:
```js
const nextConfig = {
  images: { formats: ['image/webp'] },
  redirects: async () => [
  ...
     {
      source: '/posts/:slug*',
      destination: '/410',
      permanent: true,
    },
  ...jobsRedirects,
  ]
```

**Review by miedziak — CHANGES_REQUESTED** [review]

File: next.config.js (hunk 1 above)
Comment (m-ata) [reply next.config.js]: Done!

Comment (m-ata) [reply next.config.js]: Nice Idea. I implemented that way.

File: lib/utils.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
+const redirectJobs = [
```
Comment (miedziak) [inline lib/utils.js:1]: suggestion: utils sound more like a function purpose. I would instead go with just 'redirects.js'

**Review by miedziak — CHANGES_REQUESTED** [review]

File: lib/utils.js (hunk 2 above)
Comment (m-ata) [reply lib/utils.js]: fixed!

**Review by miedziak — APPROVED** [review]

**Pull request merged**